Tests/UnitTests/PinnableTests.swift

```diff
@@ -0,0 +1,84 @@
+//
+// PinnableTests.swift
+//
+// Copyright © 2019 Button, Inc. All rights reserved. (https://usebutton.com)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+
+import XCTest
+@testable import ButtonMerchant
+
+class PinnableTests: XCTestCase {
+    
+    func testIsEqual_sameCerts_returnsTrue() {
+        // Arrange
+        let cert1 = TestCertificates.certNamed("AmazonRootCA1")
```

Comment:
Is it necessary to test the certificates themselves? I may be misunderstanding the code here but `certNamed` was created just for the purpose of tests and isn't being used elsewhere. If so, the only thing being tested is the certificate itself, right?

Reply:
It's just verifying that the `SecCertificate` object we create from the PEM string is the same as the DER format loaded from resources. Just a sanity check on the string itself, really.

The `TestCertificates` was created to avoid writing the same code to load certificates over and over.

Reply:
Aren't you just loading the same cert twice here? Where are you comparing it to the string?

Reply:
This is testing the `Pinnable` protocol where equality is implemented. It's loading the same one, but there are two separate references to it (as there would be when evaluating an auth challenge) so the equality test is a valid one to me.